packages/components/src/styles/copyright/copyright.test.stories.ts

```diff
@@ -0,0 +1,45 @@
+import '../../solid-components';
+import { storybookDefaults, storybookHelpers, storybookTemplate } from '../../../scripts/storybook/helper';
+
+const { argTypes, parameters } = storybookDefaults('sd-copyright');
+const { overrideArgs } = storybookHelpers('sd-copyright');
+const { generateTemplate } = storybookTemplate('sd-copyright');
+
+/**
+ * Add a copyright at the bottom of a image.
+ */
+export default {
+  title: 'Styles/sd-copyright/Screenshot Test',
+  tags: ['!autodocs'],
+  component: 'sd-copyright',
+  parameters: {
+    ...parameters
+  },
+  args: overrideArgs(
+    {
+      type: 'slot',
+      name: 'default',
+      value: `<img src="./placeholders/images/generic.jpg" alt="A generic placeholder jpg" class="aspect-video object-cover"/>`
+    },
+    {
+      '--copyright': '© 2024 Solid Design System'
+    }
+  ),
+  argTypes
+};
+
+export const Default = {
+  parameters: {
+    controls: {
+      disable: true
+    }
+  },
+  render: (args: { [k: string]: any }) => {
+    return generateTemplate({
+      options: {
+        templateContent: `<div class="%CLASSES% max-w-xl" style="--copyright: '${args['--copyright']}';">%SLOT%</div>`
+      },
+      args
+    });
+  }
+};
```

Comment:
I'm missing documentation of `--copyright` here. I know, that this already to questions in a team, there it would be good to explicitly mention it again.

Furthermore the information would help that you can wrap it around any element and that the text will show up at the right position.

So maybe something like:
> Wrap any element with the class and use the `--copyright` CSS custom property to set the text.

I'm pretty sure, that you will find a better solution, but you know which direction I'm thinking. :)

Reply:
What about:

>Use the CSS property `--copyright` to set the text inside a component using the class `sd-copyright`.

Reply:
That's not correct. The class with `sd-copyright` has to surround the element, that should get the copyright. E. g.

```html
<div class="sd-copyright" style="--copyright: '© 2024 Solid Design System';">
  <img src="whatever">
</div>
```

From the description you wrote it sounds more like the following should work:
```html
<img
  class="sd-copyright"
  style="--copyright: '© 2024 Solid Design System';"
  src="whatever"
>
```

But it doesn't.

Reply:
> Use the CSS property `--copyright` and the class `sd-copyright` in the parent of any element to set a copyright text